static/swagger/api_v3.json

```diff
@@ -2,10 +2,10 @@
   "swagger": "2.0",
   "info": {
     "description": "# Overview \n\n Information and statistics about FIRST Robotics Competition teams and events. \n\n# Authentication \n\nAll endpoints require an Auth Key to be passed in the header `X-TBA-Auth-Key`. If you do not have an auth key yet, you can obtain one from your [Account Page](/account). \n\n A `User-Agent` header may need to be set to prevent a 403 Unauthorized error.",
-    "version": "3.03.1",
+    "version": "3.03.2",
```

Comment:
This shouldn't bump the point release, but the minor release instead, as this does add additional fields to the models. (See `x-version-info` for rationale)

Reply:
Resolved in the latest commit.